src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/UnixUnicastIPAddressInformation.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.NetworkInformation
+{
+    /// Provides information about a network interface's unicast address
+    internal class UnixUnicastIPAddressInformation : UnicastIPAddressInformation
+    {
+        private readonly IPAddress _address;
+        private readonly IPAddress _netMask;
+
+        public UnixUnicastIPAddressInformation(IPAddress address, IPAddress netMask)
+        {
+            _address = address;
+            _netMask = netMask;
+        }
+
+        public override IPAddress Address { get { return _address; } }
+
+        /// Gets a bool value that indicates whether the Internet Protocol (IP) address is legal to appear in a Domain Name System (DNS) server database.
+        public override bool IsDnsEligible { get { throw new PlatformNotSupportedException(); } }
+
+        /// Gets a bool value that indicates whether the Internet Protocol (IP) address is transient.
+        public override bool IsTransient { get { throw new PlatformNotSupportedException(); } }
+
+        /// [To be supplied.]
```

Comment:
These comments should either be replaced with the MSDN documentation or removed.